### DISCUSSION TITLE
how to set category granularity permission?

### DISCUSSION THREAD

**Discussion by movelikeriver** [inventree/InvenTree#7124] [Q&A]
given:
```
group-a: user1, user2
group-b: user3, user4, user5

category-a: part1, part2
category-b: part3
```

currently, all users can see all parts under all categories, i'd like to achieve:
`group-a` can only manage parts under `category-a`
`group-b` can only manage parts under `category-b`

any way to config that?

**Comment by SchrodingersGat** [comment]
@movelikeriver the closest we can support currently is documented here: https://docs.inventree.org/en/stable/stock/owner/?h=ownership#stock-ownership

So, this level of granularity does not exist current for parts / part categories.

If this is something you wanted, it would take a significant amount of development effort. Perhaps you want to contribute this, or fund one of the developers?
